SpringMobileProject/app/src/test/java/com/example/mobiledevproject/dto/Parts.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.mobiledevproject.dto
+
+data class Parts( var GPU: String, var processor :String, var fan : String ){
+    override fun toString(): String {
+        return "$GPU $processor $fan"
+    }
+}
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.